### ISSUE TITLE
issue downloading audiobook

### ISSUE THREAD

**Issue by vitimilocity** [audiamus/BookLibConnect#130]
when i try to download a audiobook, it immediately stops trying to download after a second and i get a red download icon next to the book im trying to get.
![Screenshot 2023-11-01 212224](https://github.com/audiamus/BookLibConnect/assets/20304369/2772750f-a1b3-42d1-bd31-de70384b2690)

**Comment by audiamus** [comment]
The log file and the associated json response file may tell more. You'll find them the the `log` and `tmp` sub-directories of `C:\Users\<USER>\AppData\Local\audiamus\BookLibConnect`.

**Comment by audiamus** [comment]
Closing this for now. Please reopen once you can provide more information.

**Comment by Hoempi** [comment]
I got the same. I purchased three books today and two are working fine, whilst the third is showing the problem. 

I deleted all log and json files and re-downloaded all three of them. My aim was to provide examples of the same account and purchasing date working and failing. 

But: after deleting all the files it worked flawlessly. I do not know if there is a possible connection, but thought I'll report anyway.
